onebusaway-android/src/main/java/org/onebusaway/android/io/elements/ObaRegionElement.java

```diff
@@ -256,6 +305,7 @@ public String toString() {
                 ", obaBaseUrl='" + obaBaseUrl + '\'' +
                 ", siriBaseUrl='" + siriBaseUrl + '\'' +
                 ", bounds=" + Arrays.toString(bounds) +
+                ", open311Servers=" + Arrays.toString(open311Servers) +
```

Comment:
Good catch - I think you need to override `Bounds.toString()` and `Open311Servers.toString()`.